backend/app/assets/javascripts/spree/backend/components/admin_nav.js

```diff
@@ -0,0 +1,27 @@
+Spree.ready(function() {
+  toggleTooltips();
+  if (window.screen.width <= 1024 && !document.cookie.includes("admin_nav_hidden")) {
+    // Set default nav to collapse on small screens - but don't override user preference
+    document.body.classList.add("admin-nav-hidden");
+    document.cookie = "admin_nav_hidden=true; expires=Fri, 31 Dec 9999 23:59:59 GMT";
```

Comment:
Any reason not to use LocalStorage instead? 
Browser support is pretty wide https://caniuse.com/#feat=namevalue-storage
I would also suggest namespacing the key with `solidus.` or `solidus/`, e.g. `window.localStorage.get('solidus.minimize-admin-nav')`

Reply:
I believe I was initially using LocalStorage, but there was some feedback on that decision from either @tvdeyen or @seand7565. I do not remember it 🤦‍♂